src/main/java/com/mcmoddev/alt/AdditionalLootTables.java

```diff
@@ -8,89 +9,100 @@
 
 import net.minecraft.util.ResourceLocation;
-import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.LootTableLoadEvent;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.DistExecutor;
 import net.minecraftforge.fml.common.Mod;
-import net.minecraftforge.fml.common.Mod.EventHandler;
-import net.minecraftforge.fml.common.Mod.Instance;
-import net.minecraftforge.fml.common.SidedProxy;
-import net.minecraftforge.fml.common.event.FMLFingerprintViolationEvent; 
-import net.minecraftforge.fml.common.event.FMLInitializationEvent;
-import net.minecraftforge.fml.common.event.FMLInterModComms;
-import net.minecraftforge.fml.common.event.FMLPostInitializationEvent;
-import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
-import net.minecraftforge.fml.common.event.FMLServerStartingEvent;
+import net.minecraftforge.fml.event.lifecycle.FMLCommonSetupEvent;
+import net.minecraftforge.fml.event.lifecycle.InterModProcessEvent;
+import net.minecraftforge.fml.event.server.FMLServerStartingEvent;
+import net.minecraftforge.fml.event.server.FMLServerStoppedEvent;
+import net.minecraftforge.fml.loading.FMLPaths;
+
 
-@Mod(modid = AdditionalLootTables.MODID,
-	version = AdditionalLootTables.VERSION,
-	name = AdditionalLootTables.NAME,
-	acceptedMinecraftVersions = "[1.12,)",
-	certificateFingerprint = "@FINGERPRINT@")
+@Mod(AdditionalLootTables.MODID)
 public class AdditionalLootTables {
-	public static final String NAME = "Additional Loot Tables";
 	public static final String MODID = "alt";
 	public static final String VERSION = "2.0.3";
 
-	@Instance
 	public static AdditionalLootTables INSTANCE = null;
 	public static Config config = null;
 	public static Logger logger = LogManager.getFormatterLogger(MODID);
 	public static final String ALT_FOLDER_NAME = "additional-loot-tables";
 	private static Path lootFolder;
-    private static final String PROXY_BASE = "com.mcmoddev." + MODID + ".proxy.";
     private static final PluginLoader pluginLoader = new PluginLoader();
-    
-    @SidedProxy(clientSide = PROXY_BASE + "ClientProxy", serverSide = PROXY_BASE + "ServerProxy")
-    public static CommonProxy proxy;
 
-	@EventHandler
-	public void onFingerprintViolation(FMLFingerprintViolationEvent event) {
-		logger.warn("Invalid fingerprint detected!");
-	}
+    public static CommonProxy proxy = DistExecutor.runForDist(() -> () -> new ClientProxy(), () -> () -> new CommonProxy());
 
-	@EventHandler
-	public static void preInit(FMLPreInitializationEvent event) {
-		final String ALT_BASE_CONFIG_PATH = event.getSuggestedConfigurationFile().getParent();
-		lootFolder = Paths.get(ALT_BASE_CONFIG_PATH, ALT_FOLDER_NAME);
-		config = new Config(event);
+	public AdditionalLootTables() {
+		if (INSTANCE == null)
+			INSTANCE = this;
+		else
+			throw new RuntimeException("Duplicated Class Instantiation: CustomItems");

```

Comment:
Copypasta ?

Reply:
Yes /w\ It was from another mod I just ported to 1.15.2

Reply:
Fix in later commits